### PR TITLE
feat(Box): elevation and border

### PR DESCRIPTION
**Pull request by landbot-dev**

### CODE REVIEW

**Review by amalv — APPROVED** [review]

File: src/Box/utils.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { css } from 'styled-components';
+import { getColorFromTheme } from '../shared/mixins';
+import { Theme } from '../shared/theme.types';
+import { BORDER_RADIUS_MAPPING, BOX_SHADOW_MAPPING } from './constants';
+import { StyledBoxProps } from './types';
+
+export const getElevationStyles = ({ $elevation = 0 }: StyledBoxProps) => {
```
Comment (amalv) [inline src/Box/utils.ts:7]: Shorter alternative:

```
export const getElevationStyles = ({ $elevation = 0 }: StyledBoxProps) =>
  css`
    box-shadow: ${BOX_SHADOW_MAPPING[$elevation]};
  `;
  ```

Comment (EduardoRG) [reply src/Box/utils.ts]: Changed!

File: src/Box/utils.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { css } from 'styled-components';
+import { getColorFromTheme } from '../shared/mixins';
+import { Theme } from '../shared/theme.types';
+import { BORDER_RADIUS_MAPPING, BOX_SHADOW_MAPPING } from './constants';
+import { StyledBoxProps } from './types';
+
+export const getBackgroundStyles = ({ $background, theme }: StyledBoxProps & { theme: Theme }) => {
+  if ($background) {
+    return css`
+      background-color: ${getColorFromTheme(theme, $background)};
```
Comment (EduardoRG) [inline src/Box/utils.ts:10]: @david-gomey @solilokiam @amalv I'm not sure if `background` should be the name of the prop or `backgroundColor`, to correspond the css rule. What do you think?

Comment (david-gomey) [reply src/Box/utils.ts:10]: `backgroundColor` would be nice, in my opinion

Comment (EduardoRG) [reply src/Box/utils.ts]: Agree, changed!

**Pull request merged**